sktime/forecasting/compose/_pipeline.py

```diff
@@ -1378,6 +1407,19 @@ def __init__(
         # }
         # self.set_tags(**tag_translate_dict)
 
+        if (
+            self.fit_behaviour == "use_forecast"
+            and self.predict_behaviour == "use_actuals"
+        ):
+            warn(
```

Comment:
this should have `self` as arg, or it will not be affected by `set_config`

Reply:
and stacklevel=2?

Reply:
I'll add stacklevel (I thought that was default), but did not understand the first comment on `self`. I haven't used this utility function from `sktime` before. Can you please explain what do I need to change and how will it affect configuration?